@AH-64D Apache Longbow/Addons/fza_ah64_systems/headers/systems.hpp

```diff
@@ -21,7 +21,7 @@
 #define SYS_MIN_ACC_PSI   1650
 #define SYS_HYD_MIN_LVL   0.1
  
-#define SYS_BATT_TIMER    3.0   //min
+#define SYS_BATT_TIMER    12.0  //min
```

Comment:
This should really be a separate PR due to it not being on-topic, but will ignore it this time